.gitignore

```diff
@@ -9,6 +9,7 @@
 *.o
 *.obj
 *.od
+*.ssexwsp
```

Comment:
I see. You don't have to remove it. However, it is not popular yet, so could write the description to the log of the commit that changes .gitignore.